packages/components/src/Combobox/components/ComboboxActivator/ComboboxActivator.tsx

```diff
@@ -15,10 +15,10 @@ export function ComboboxActivator(props: ComboboxActivatorProps) {
   ) {
     return React.cloneElement(props.children, {
       role: accessibilityAttributes.role,
-      onClick: toggleOpen,
+      onClick: handleOpen,
     });
   } else if (typeof props.children === "function") {
-    return props.children({ ...accessibilityAttributes, toggleOpen });
+    return props.children({ ...accessibilityAttributes, open: handleOpen });
```

Comment:
I like `open()` better than `handleOpen()` when considering this will be used by a consumer.